blockchain/blockdao/blockdao_test.go

```diff
@@ -442,6 +441,8 @@ func BenchmarkBlockCache(b *testing.B) {
 		defer func() {
 			require.NoError(b, os.RemoveAll(testPath))
```

Comment:
L442-L443 is unnecessary due to L444-L445

Reply:
yes removed